libs/ballot-interpreter/src/hmpb-rust/timing_marks.rs

```diff
@@ -705,6 +803,38 @@ pub enum FindCompleteTimingMarksError {
     /// Not enough corners of the ballot card could be found.
     #[error("Not enough corners of the ballot card could be found")]
     MissingCorners,
+
+    /// One of the timing mark border sides is invalid.
+    #[error("Invalid timing mark side: {side_marks:?}")]
+    InvalidTimingMarkSide { side_marks: SideMarks },
+}
+
+#[derive(Debug)]
+pub enum SideMarks {
+    #[allow(dead_code)]
```

Comment:
This is here because we only ever debug-print this `enum` and don't actually access the variant properties.